storage/client/blob/src/test/java/com/azure/storage/blob/LargeFileTest.java

```diff
@@ -9,7 +12,7 @@
 
 public class LargeFileTest {
     private static final Random RANDOM = new Random();
-    private static final String filePath = "C:\\Users\\jianghlu\\10g.dat";
```

Comment:
Really weird to see person's name here

Reply:
I noticed that too, looks like some testing unit tests made their way into the central repo, shouldn't be an issue just looks weird.